.github/CONTRIBUTING.md

```diff
@@ -58,3 +59,111 @@ Things we pay attention in a PR :
 
 
 _If you are unsure about git you can have a look on our [git tips to have a clean history](https://github.com/mockito/mockito/wiki/Using git to prepare your PR to have a clean history)._
+
+
+## Coding style
+
+This section is not about some kind of fruitless tabs vs spaces debate. It's about having the code readable, the project grows and it is not rare to read contributions from many different individuals. Each one of us has a different writing style, we are fine with this. Without enforcing we may be picky about it, however this we think this may improve helps readability for everyone.
+
+_This includes IntelliJ IDEA instructions, however we are sure there's similar settings in all major IDEs._
+
+But first of all, make sure that : 
+
+* Don't use tabs, only spaces
+* Character encoding is **UTF-8**
+* Line ending character is unix-style **`LF`**
+* New line is added at end of file: `IntelliJ setting > Editor > General > Ensure line feed at file end on save`
+
+### Imports
+
+Imports must be sorted in the following order :
+
+1. `import java.*`
+2. `import javax.*`
+3. `import all other imports`
+4. blank line
+5. `import static java.*`
+6. `import static javax.*`
+7. `import static all other imports`
```

Comment:
I would actually place static imports before normal imports. Any particular reason for this order?